app/models/attribute/ConfigTable.scala

```diff
@@ -0,0 +1,204 @@
+package models.attribute
+
+import models.daos.slick.DBTableDefinitions.{DBUser, UserTable}
+import models.utils.MyPostgresDriver.simple._
+import play.api.Play.current
+import scala.slick.lifted.ForeignKeyQuery
+
+case class ConfigApi(apiAttributeCenterLat: Double, apiAttributeCenterLng: Double,
```

Comment:
Since it's useful to add a `MapParams` class to simplify other code, it might make sense to replace this big `ConfigAPI` class with that one?

Reply:
Kinda agree? If you mean just re-naming `ConfigAPI` to `MapParams` then that works! However, adding additional parameters would be interesting since `ConfigAPI` already has 21 parameters and if I recall correctly there is a 22 parameter limit.

Reply:
I'm suggesting removing the `ConfigApi` class, and instead adding in a `MapParams` class that has only 7 (I think it was 7?) fields, but the `Config` class would contain 3-4 of them instead of just the one `ConfigAPI` class.

Reply:
It'd be something like
```
case class Config(
    openStatus: String, mapathonEventLink: Option[String], overallMapParams: MapParams,
    tutorialStreetEdgeID: Int, offsetHours: Int, excludedTags: String,
    apiAttributeParams: MapParams, apiStreetParams: MapParams, apiRegionParams: MapParams
)
```

Reply:
Sounds good to me, will be pinging you on Slack about this later then :)